### PR TITLE
Adds new plugin [marcomow/ble-bulb-card]

### PR DESCRIPTION
**Pull request by marcomow**

### CODE REVIEW

**Comment by marcomow** [comment]
Hi @ludeeus, as this is my first time ever issuing a pull request, I don't really understand what should I do to fix the faulty checks. Can you kindly tell me what should I work on?

File: plugin [hunk 1]
```diff
@@ -128,6 +128,7 @@
   "lukevink/lovelace-buien-rain-card",
   "madmicio/channel-pad",
   "madmicio/LG-WebOS-Remote-Control",
+  "marcomow/ble-bulb-card"
```
Comment (ludeeus) [inline plugin:131]:
```suggestion
  "marcomow/ble-bulb-card",
```

Not valid JSON

**Review by ludeeus — CHANGES_REQUESTED** [review]

**Comment by marcomow** [comment]
hi @ludeeus just checking out as this is my first pull request - am I missing anything?

**Comment by KTibow** [comment]
@marcomow 
> DO NOT REQUEST REVIEWS, THAT IS JUST RUDE, IF YOU DO THE PULL REQUEST WILL BE CLOSED!

They just need to get around to it, also some people have different time zones you know.

**Comment by marcomow** [comment]
@KTibow sorry, I didn't mean to be rude, I was genuinely asking as this is my first experience with a pull request.

**Comment by KTibow** [comment]
It’s okay. It’s just that this is a tedious process, and not every PR can be reviewed.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Pull request merged**